docs/pages/patterns/combobox-input.md

```diff
@@ -34,22 +34,57 @@ If the entries are not validated by the application, users can also enter custom
       </div>
   </div>
   <div class="fd-popover__body fd-popover__body--no-arrow fd-popover__body--dropdown fd-popover__body--dropdown-fill" aria-hidden="true" id="F4GcX348">
-        <ul class="fd-list fd-list--dropdown" role="listbox">
-            <li role="option" tabindex="0" class="fd-list__item is-selected">
-                <span class="fd-list__title">
-                    <span class="fd-list__bold">App</span>le
-                </span>
-            </li>
-            <li role="option" tabindex="0" class="fd-list__item">
-                <span class="fd-list__title">Orange</span>
-            </li>
-            <li role="option" tabindex="0" class="fd-list__item">
-                <span class="fd-list__title">Banana</span>
-            </li>
-            <li role="option" tabindex="0" class="fd-list__item">
-                <span class="fd-list__title">Kiwi</span>
-            </li>
-        </ul>
+    <div class="fd-popover__wrapper docs-max-height">
```

Comment:
here I still see the issue, which btw in master is not there
![Screen Shot 2020-05-22 at 4 24 35 PM](https://user-images.githubusercontent.com/4380815/82707006-2bd2c380-9c49-11ea-93fb-3c229cacda91.png)

Reply:
I think we agreed to keep it as it is right now, there is no way to hide this part of scrollbar. Other solutions (hiding whole/customizing) were abandoned.